flink-connectors/flink-connector-jdbc/src/main/java/org/apache/flink/connector/jdbc/table/JdbcLookupFunction.java

```diff
@@ -177,7 +178,7 @@ public void eval(Object... keys) {
 				}
 
 				try {
-					if (!dbConn.isValid(CONNECTION_CHECK_TIMEOUT_SECONDS)) {
+					if (!dbConn.isValid(connectionCheckTimeoutSeconds)) {
```

Comment:
We should also add property in `JdbcTableSourceSinkFactory` and add test in `JdbcTableSourceSinkFactoryTest`, for this is a legacy lookup function. You can refer to the commit I provided.

Reply:
yes,it is necessary,i will add it

Reply:
hi,@wangxlong .i think  add property in JdbcTableSourceSinkFactory and add test in JdbcTableSourceSinkFactoryTest is necessary,but is not for lookup function.as i know  is should be test in testJdbcCommonProperties()

Reply:
Sorry, you are right. And add a invalid value which is less than 1 second in `testJdbcValidation`.